scripts/check_calendars.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
```

Comment:
Move this to the `bin` directory instead. The `scripts` dir is for scripts that are to be run via a management command.